katello_certs_tools/katello_ssl_tool.py

```diff
@@ -249,6 +249,17 @@ def genPublicCaCert(password, d, verbosity=0, forceYN=0):
         if err:
             print "STDERR:", err
 
+    if d['--other-ca-certs']:
+        if verbosity >= 0:
+            print "\nAdding other public CA certificates"
+        with open(cleanupAbsPath(ca_cert), 'a') as outfile:
+            for fname in d['--other-ca-certs'].split(','):
+                with open(fname) as infile:
+                    for line in infile:
+                        outfile.write(line)
+                    infile.close()
```

Comment:
Since you're using the contextmanager to open the file it will autoclose and this is redundant.